apps/extension/src/routes/popup/home/frontend-link.tsx

```diff
@@ -0,0 +1,26 @@
+import { useStore } from '../../../state';
+import { getDefaultFrontend } from '../../../state/default-frontend';
+import { Button } from '@penumbra-zone/ui/components/ui/button';
+import { ExternalLink } from 'lucide-react';
+import { MouseEventHandler } from 'react';
+import { usePopupNav } from '../../../utils/navigate';
+import { PopupPath } from '../paths';
+
+export const FrontendLink = () => {
+  const frontendUrl = useStore(getDefaultFrontend);
+  const navigate = usePopupNav();
+
+  const onClick: MouseEventHandler = event => {
+    if (frontendUrl) return;
```

Comment:
I think I'm confused by the `<a>` and `onClick` usage below. This is the case where a default frontend is not set right?

Would you be able to add some code comments so it's clearer for folks?